services/login_service/app/.gitignore

```diff
@@ -0,0 +1,5 @@
+__pycache__/
+auth/__pycache__/
+config/__pycache__/
+email_verification/__pycache__/
+verify_table/__pycache__/
```

Comment:
```
*.pyc
*.pyo
```
would be better so it won't pickup any of this files. Even better, there is github template for gitignore
```
# Byte-compiled / optimized / DLL files
__pycache__/
*.py[cod]

# C extensions
*.so

# Distribution / packaging
bin/
build/
develop-eggs/
dist/
eggs/
lib/
lib64/
parts/
sdist/
var/
*.egg-info/
.installed.cfg
*.egg

# Installer logs
pip-log.txt
pip-delete-this-directory.txt

# Unit test / coverage reports
.tox/
.coverage
.cache
nosetests.xml
coverage.xml

# Translations
*.mo

# Mr Developer
.mr.developer.cfg
.project
.pydevproject

# Rope
.ropeproject

# Django stuff:
*.log
*.pot

# Sphinx documentation
docs/_build/
```